mssqlcli/localized_strings.py

```diff
@@ -16,4 +16,4 @@ def translation(domain=DOMAIN, localedir=LOCALE_DIR, languages=None):
 
 ## Localized Strings
 def goodbye():
-    return _(u'Goodbye!')
\ No newline at end of file
+    return _(u'Goodbye!')
```

Comment:
Let's revert this - it's already been changed and no point in changing it again.

Reply:
You asked me to add newLine in previous PR. And I added it in this PR.

Reply:
This is your comment in the previous PR.
![image](https://user-images.githubusercontent.com/19577035/64292493-61027e80-cf1f-11e9-9f2c-96d5a2e7dd3d.png)